controllers/builders/roles.go

```diff
@@ -0,0 +1,59 @@
+package builders
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```

Comment:
👍🏼   This is great.

Reply:
TBH I will have to revert this as I was blidnly assuming executor is the same as everything else.  

Binoculars and Executor do have different ClusterRoles so I will revert this but ty!